packages/services/src/services/Translation/__tests__/Translation.test.js

```diff
@@ -98,6 +98,28 @@ describe('TranslationAPI', () => {
     expect(response).toEqual(responseSuccess);
   });
 
+  it('should fetch the i18n data from given endpoint', async () => {
```

Comment:
Not sure if this is more ambitious, but might be nice if we could test what is being set in sessionStorage, and if it's either using the default or using a custom key. We'd probably need to mock session storage though in our unit tests: https://www.npmjs.com/package/mock-local-storage

Reply:
Used the mock session storage that was already set up to test